File: Extensions/ConsoleInput.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Extensions
{
    public static class ConsoleInput
    {
        public static void SafeIntInput(string message, out int value)
        {
            while (true)
            {
                Console.Write(message + ": ");


                if (int.TryParse(Console.ReadLine(), out value))
                    break;
                else
                    Console.WriteLine("Введіть ціле число!");
            }
        }

        public static void SafeIntInput(string message, out double value)
        {
            while (true)
            {
                Console.Write(message + ": ");


                if (double.TryParse(Console.ReadLine(), out value))
                    break;
                else
                    Console.WriteLine("Введіть ціле число!");
            }
        }
    }
}

File: Laba2-Python/ConsoleInput.py
def SafeIntInput(message):
    while True:
        try:
            value = int(input(message + ": "))
            return value
        except ValueError:
            print("Введіть ціле число!")

def SafeFloatInput(message):
    while True:
        try:
            value = float(input(message + ": "))
            return value
        except ValueError:
            print("Введіть ціле число!")